Remove Assist command execution and chat feature

jakule opened the pull request:
The changes remove tests and related code for command execution in the assist package and web command handling.

changelog: Remove Assist chat API.

jakule committed:
Remove Assist command execution and chat feature

The changes remove tests and related code for command execution in the assist package and web command handling.

r0mant reviewed (approved):
@jakule Can you add a section to "breaking changes" in changelog in this PR as well?

jakule committed:
Remove Teleport Assist chat from CHANGELOG.md

The changes in this commit reflect the removal of the Teleport Assist chat from the CHANGELOG documentation of Teleport version 16. The Assist feature continues to be available in the SSH Web Terminal and Audit Monitoring.

jakule committed:
Remove assist chat from the test plan

jakule committed:
Merge branch 'master' into jakule/remove-assist-chat-api
